aws/autoscaling.go

```diff
@@ -350,12 +356,16 @@ func (a *AutoScalingGroup) Filter(filter filters.Filter) bool {
 		}
 	case "CreatedTimeInTheLast":
 		d, err := time.ParseDuration(filter.Arguments[0])
-		if err == nil && time.Since(a.CreatedTime) < d {
+		if err == nil && a.CreatedTime != nil && time.Since(*a.CreatedTime) < d {
 			matched = true
 		}
 	case "CreatedTimeNotInTheLast":
 		d, err := time.ParseDuration(filter.Arguments[0])
-		if err == nil && time.Since(a.CreatedTime) > d {
+		if err == nil && a.CreatedTime != nil && time.Since(*a.CreatedTime) > d {
```

Comment:
Using the embedded struct's CreatedTime, which is a pointer, so need to do a nil check